api/envoy/api/v3alpha/core/backoff.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoy.api.v3alpha.core;
```

Comment:
We don't allow any protos in `envoy.api.v3alpha.core`. See what was done for this file's peers in v2 with `option (udpa.annotations.file_migrate).move_to_package`